plugins/server/com.ucasoft/simple-cache/versions.ktor.yaml

```diff
@@ -0,0 +1 @@
+"[2.0,)":ktor-simple-cache:==
```

Comment:
The right side version should point to your artifact here:

```
"[2.0,)": com.ucasoft.ktor:ktor-simple-cache:0.+
```

The `==` symbol is only for artifacts that are synced with the Ktor version of the generated project (i.e. plugins inside the ktor repository).